servatrice/migrations/servatrice_0024_to_0025.sql

```diff
@@ -0,0 +1,6 @@
+-- Servatrice db migration from version 24 to version 25
+
+ALTER TABLE cockatrice_uptime ADD COLUMN mods_count int(11) NOT NULL DEFAULT 0;
+ALTER TABLE cockatrice_uptime ADD COLUMN mods_list TEXT;
+
+UPDATE cockatrice_schema_version SET version = 25 WHERE version = 24;
```

Comment:
The checker script expects the exact string:
```
UPDATE cockatrice_schema_version SET version=25 WHERE version=24;
```
Please remove spaces around the equal signs, we'll fix the script later.